packages/backend-defaults/api-report.md

```diff
@@ -0,0 +1,19 @@
+## API Report File for "@backstage/backend-defaults"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AnyServiceFactory } from '@backstage/backend-plugin-api';
+import { Backend } from '@backstage/backend-app-api';
+
+// @public (undocumented)
+export function createBackend(options?: CreateBackendOptions): Backend;
+
+// @public (undocumented)
+export interface CreateBackendOptions {
+  // (undocumented)
+  serviceFactories?: AnyServiceFactory[];
+}
+
+// (No @packageDocumentation comment for this package)
```

Comment:
maybe keep this one free from even small notes like this and (undocumented) from the get-go? :)